efi/secureboot_namespace_rules.go

```diff
@@ -19,8 +19,120 @@
 
 package efi
 
-import "crypto/x509"
+import (
+	"bytes"
+	"crypto/x509"
+
+	"golang.org/x/xerrors"
+)
 
 type vendorAuthorityGetter interface {
 	VendorAuthorities() ([]*x509.Certificate, error)
 }
+
```

Comment:
this from previous PRs need doc comments

Reply:
I've added a comment to this now.